package/koreader/package

```diff
@@ -2,15 +2,15 @@
 pkgname=koreader
 pkgdesc="KOReader"
 url=https://github.com/koreader/koreader
-pkgver=2020.08.01-1
-timestamp=2020-09-19T03:30Z
+pkgver=2020.09-1
+timestamp=2020-09-25T01:42Z
 section=utils
```

Comment:
Maybe this should go into the `readers` section? See [the `section` documentation](https://github.com/toltec-dev/toltec/blob/testing/docs/package.md#section-required).